TheFourFifths/consus-client
# TFF 77: Invalid student id

### Summary

Prompts an error modal when an invalid/missing student id is scanned.

Note: I think the PR should work as-is, but the modal won't appear until TFF-71 is completed and merged. We should probably wait until that story is merged into dev before this PR gets merged so that we can verify it actually works.

This also requires the [Sister PR to consus](https://github.com/TheFourFifths/consus/pull/16) to work.

### Checklist


Finally, is it ready to be reviewed and merged: yes :frog: 

### How to Review

1. Run the server locally, using the changes from the [sister PR](https://github.com/TheFourFifths/consus/pull/16).
2. Start the client.
3. Enter a 6-digit id that does not correlate to a student yet (e.g. 999999).
4. Verify that a modal pops up with an appropriate error message.

### Links

- [TFF-77](https://msoese.atlassian.net/browse/TFF-77)
- [Sister PR to consus](https://github.com/TheFourFifths/consus/pull/16)
